### PR TITLE
Adds integration tests for language, user(group) and contenttype

### LINKED ISSUE #40: Integrations tests for languages, contenttypes and user(groups) import

### PR DESCRIPTION
**Pull request by htollefsen**
Resolves #40.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -14,6 +14,9 @@
         <testsuite name="Transfer eZ Platform Unit Tests">
             <directory suffix="Test.php">./tests/Transfer/EzPlatform</directory>
         </testsuite>
+        <testsuite name="Transfer eZ Platform Integration Tests">
+            <directory suffix="Test.php">./tests/integrational</directory>
```
Comment (ghost) [inline phpunit.xml.dist]: Let's use `integration`

File: tests/bootstrap.php [hunk 2]
```diff
@@ -18,3 +18,4 @@
 $loader = require __DIR__.'/../vendor/autoload.php';
 $loader->addPsr4('Transfer\\EzPlatform\\', __DIR__.'/../src/Transfer/EzPlatform');
 $loader->addPsr4('Transfer\\EzPlatform\\Tests\\', __DIR__.'/Transfer/EzPlatform/Tests');
+$loader->addPsr4('Integrational\\', __DIR__.'/Integrational');
```
Comment (ghost) [inline tests/bootstrap.php]: We don't need to autoload the tests, as PHPUnit does not rely on composer's autoload.

File: tests/integrational/ContenttypeTest.php [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests\integrational;
```
Comment (ghost) [inline tests/integrational/ContenttypeTest.php]: Namespace should be uppercase and with proper prefix: `namespace Transfer\EzPlatform\Tests\Integration;`

File: tests/integrational/UsergroupTest.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace tests\integrational;
+
+use Transfer\Adapter\Transaction\Request;
+use Transfer\EzPlatform\Adapter\EzPlatformAdapter;
+use Transfer\EzPlatform\Data\UserGroupObject;
+use Transfer\EzPlatform\Tests\EzPlatformTestCase;
+
+class UsergroupTest extends EzPlatformTestCase
```
Comment (ghost) [inline tests/integrational/UsergroupTest.php]: How about `UserGroupTest`?

File: tests/integrational/ContenttypeTest.php [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use Transfer\Adapter\Transaction\Request;
+use Transfer\EzPlatform\Adapter\EzPlatformAdapter;
+use Transfer\EzPlatform\Data\ContentTypeObject;
+use Transfer\EzPlatform\Tests\EzPlatformTestCase;
+
+class ContenttypeTest extends EzPlatformTestCase
```
Comment (ghost) [inline tests/integrational/ContenttypeTest.php]: How about `ContentTypeTest`?

File: tests/integrational/ContenttypeTest.php (hunk 3 above)
Comment (htollefsen) [inline tests/integrational/ContenttypeTest.php]: Actually, I took a look at the test-namespaces in [transfer/transfer TransferTestCase](https://github.com/transfer-framework/transfer/blob/1.0/tests/functional/TransferTestCase.php).
Anyway, should I also structure it in the suggested namespace path? I must have misunderstood our talk earlier this week.

Comment (ghost) [inline tests/integrational/ContenttypeTest.php]: I think something like this would suffice: https://github.com/transfer-framework/ezplatform/pull/44/files

Comment (ghost) [inline tests/integrational/ContenttypeTest.php]: I had not noticed myself that the namespace in  [transfer/transfer TransferTestCase](https://github.com/transfer-framework/transfer/blob/1.0/tests/functional/TransferTestCase.php) does not have the proper case. I'll note that.

**Pull request merged**